<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorContactGroup</class>
 <widget class="QWidget" name="EditorContactGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="groupName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Recipient</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="recipient1"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="recipient2"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="addRecipientButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="Akonadi::CollectionComboBox" name="collectionSelector"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Akonadi::CollectionComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">akonadi/collectioncombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
